esp/src/src-react/components/Monitoring.tsx

```diff
@@ -0,0 +1,144 @@
+import * as React from "react";
+import { CommandBar, ContextualMenuItemType, ICommandBarItemProps, Image, Link } from "@fluentui/react";
+import { MachineService } from "@hpcc-js/comms";
+import * as domClass from "dojo/dom-class";
+import { ShortVerticalDivider } from "./Common";
+import { tree } from "./DojoGrid";
+import { useFluentGrid } from "../hooks/grid";
+import { HolyGrail } from "../layouts/HolyGrail";
+import * as Utility from "src/Utility";
+import nlsHPCC from "src/nlsHPCC";
+
+function getStatusImageName(row) {
+    switch (row.Status) {
+        case "Normal":
+        default:
+            return "normal.png";
```

Comment:
By convention default should always be at the bottom of the switch statement (in c++ this would be an error)